compiler/noirc_frontend/src/parser/parser/path.rs

```diff
@@ -25,7 +25,7 @@ fn empty_path() -> impl NoirParser<Path> {
     let make_path = |kind| move |_, span| Path { segments: Vec::new(), kind, span };
     let path_kind = |key, kind| keyword(key).map_with_span(make_path(kind));
 
-    choice((path_kind(Keyword::Crate, PathKind::Crate), path_kind(Keyword::Dep, PathKind::Dep)))
+    choice((path_kind(Keyword::Crate, PathKind::Crate), path_kind(Keyword::Dep, PathKind::Plain)))
```

Comment:
On line 19 there's also `path_kind(Keyword::Dep, PathKind::Dep)`... maybe that should have also been changed? Or maybe `PathKind::Dep` should be removed completely?

Reply:
IIRC, this was kept both to ensure
1. We can parse the old-style `use` statements
2. We can distinguish which was which when formatting